### PR TITLE
Adding function and test files

### LINKED ISSUE #29: Finish Function 3 - Plot
Please finish funciton 4:
- the function generates the plot of the metrics
- unit test

### PR DESCRIPTION
**Pull request by WeilinHan8**
Hi team,
I added my function content and test file for my function, Please kindly review them.

### CODE REVIEW

File: tests/test_dailyplot.py
Comment (beth-ouyang) [inline tests/test_dailyplot.py:1]: It would be better to make the file name align with the function name, so maybe add an underscore!

**Review by beth-ouyang — CHANGES_REQUESTED** [review]

**Comment by WeilinHan8** [comment]
@beth-ouyang  I have connected with Cindy (TA), she said considering the number of tests I have included in the test function and the nature of my function (creating a plot). The current test file is good enough

**Comment by beth-ouyang** [comment]
ohh that's great!!! I will approve and merge !

**Review by tangyl92 — APPROVED** [review]

**Review by beth-ouyang — APPROVED** [review]
GOOOOD!!!

**Pull request merged**